src/ape/types/trace.py

```diff
@@ -216,3 +222,26 @@ class TraceFrame(BaseInterfaceModel):
     """
     The number of external jumps away the initially called contract (starts at 0).
     """
+
+    raw: Dict
+    """
+    The raw trace frame data.
+    """
+
+
+class LineTraceNode(BaseModel):
+    source_id: str
+    """
+    The local project's contract's relative path str as an ID.
+    """
+
+    method_id: str
```

Comment:
Sometimes there are items that aren't contained within a particular method

For example, when you don't have any payable methods in Vyper, it will inline a nonpayable check (e.g. `assert msg.value == 0`) at the top of the code, and not within any particular method

Reply:
But do we want to show these ever?

Reply:
Yes this would be an important programming block to assure we had conditional coverage for

Reply:
It will be part of branch coverage.